tools/cpp/unix_cc_configure.bzl

```diff
@@ -534,7 +534,7 @@ def configure_unix_toolchain(repository_ctx, cpu_value, overriden_tools):
         ["/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk"],
     )
 
-    generate_modulemap = is_clang and not darwin
```

Comment:
This was disabled before on MacOS because it was too slow?

Reply:
correct